.github/workflows/format.yml

```
@@ -0,0 +1,23 @@
+3# This GitHub Actions workflow checks if C/C++ files are correctly formatted.
```

actionlint (1.7.7)

workflow is scalar node but mapping node is expected

(syntax-check)